src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -164,15 +170,25 @@ private void stepInner(FlightContext flightContext) throws SQLException {
             new NamedParameterPreparedStatement(connection, sqlInsertFlightLog)) {
       startTransaction(connection);
       statement.setString("flightId", flightContext.getFlightId());
+      statement.setInstant("logTime", logTime);
+
+      // TODO: Column working_parameters is being phased out in favor of table flightworking.
```

Comment:
nit: prefer to TODO(PF-NNN) a ticket for cleanup. Makes searching later easier. Ditto others